free-tunes/Controllers/TrackExplorerViewController.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  CategoriesViewController.swift
+//  Free-tunes
+//
+//  Created by Tshwarelo Mafaralala on 2022/02/28.
+//
+
+import UIKit
+import AVFoundation
+
+class TrackExplorerViewController: UIViewController {
+    
+// MARK: - Interface Builder Outlets
+    @IBOutlet private weak var trackCollectionView: UICollectionView!
+    
+    private lazy var exploreViewModel = TrackExplorerViewModel()
+    private var looper: AVPlayerLooper!
+    private var player: AVQueuePlayer!
+
+// MARK: - Runtime Methods
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        trackCollectionView.register(UINib(nibName: "TrackCollectionViewCell", bundle: .main), forCellWithReuseIdentifier: "TrackCollectionViewCell")
+        trackCollectionView.isPagingEnabled = true
+        getExploreViewContent()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        self.navigationController?.setNavigationBarHidden(true, animated: animated)
+        super.viewWillAppear(animated)
```

Comment:
Your super should always be the first thing in a method

Reply:
I had the changes locally, I must have lost them. I'll update it.

Reply:
Looks like this still needs to be addressed 😄

Reply:
I keep trying to update it but it's just not outdating the code